src/main/flight/pid.c

```diff
@@ -447,14 +447,15 @@ STATIC_UNIT_TESTED FAST_CODE_NOINLINE float pidLevel(int axis, const pidProfile_
 #endif
     angle = constrainf(angle, -levelAngleLimit, levelAngleLimit);
     const float errorAngle = angle - ((attitude.raw[axis] - angleTrim->raw[axis]) / 10.0f);
+    const float angleFF = levelAngleLimit * getRcCommandDelta(axis);
```

Comment:
this code is only correct if you are using angle mode and don't have any angle mode expo. This code is wrong if you have expo or if you are in a GPS rth home, and FF would still want to be applied in a GPS rth mode.

Reply:
Yes. I'll need to add a getLevelRcCommandDelta function to apply that. This is just for testing.